### ISSUE TITLE
Proxy DNS not work

### ISSUE THREAD

**Issue by tdzard94** [semigodking/redsocks#168]
Im just clone your project and build for android
Everything work but when i test with socks5 proxy i got result like: dns_ip:443
But i want to get result like: domain:443
What i need to do?
Thank for reply

**Comment by semigodking** [comment]
That's how transparent proxy works.

Zard ***@***.***> 于 2021年11月20日周六 18:39写道：

**Comment by tdzard94** [comment]
is your project support?
i tried anything but alway got same result

**Comment by semigodking** [comment]
No. This is limitation from transparent proxy implementation from OS.

Zard ***@***.***> 于 2021年11月20日周六 18:51写道：

**Comment by tdzard94** [comment]
thank you
